tests/k4unit/startupcycles/run_startupcycles.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+${TORQHOME}/torq.sh start discovery1
+${TORQHOME}/torq.sh start stp1
+${TORQHOME}/torq.sh start feed1
```

Comment:
I could be mistaken, but isn't the point of the test to make sure it errors as expected when feed1 is not available? why do we need to start feed1?